client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java

```diff
@@ -0,0 +1,156 @@
+package com.example.criengine;
+
+import android.view.View;
+import android.widget.EditText;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+import androidx.viewpager2.widget.ViewPager2;
+
+import com.example.criengine.Activities.LoginActivity;
+import com.example.criengine.Activities.RootActivity;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.robotium.solo.Solo;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * RootActvity test Suite
+ * Uses Robotium Framework to test UI interaction
+ */
+public class RootActivityTest {
+    private Solo solo;
+    private ViewPager2 viewPager;
+    private BottomNavigationView bottomNav;
+
+    @Rule
+    public ActivityTestRule<LoginActivity> rule =
+            new ActivityTestRule<>(LoginActivity.class, true, true);
+
+    /**
+     * Initialize solo instance before all tests
+     * @throws Exception
+     */
+    @Before
+    public void setUp() throws Exception {
+        solo = new Solo(InstrumentationRegistry.getInstrumentation(), rule.getActivity());
+        // Asserts that the current activity is the LoginActivity.
+        solo.assertCurrentActivity("Wrong Activity", LoginActivity.class);
+        // Get view for EditText and enter a city name
+        solo.enterText((EditText) solo.getView(R.id.loginEditTextEmail), "user2@email.com");
+        solo.enterText((EditText) solo.getView(R.id.loginEditTextPassword), "password");
+
+        solo.clickOnButton("Login");
+
+        // Asserts that the current activity is the RootActivity.
+        solo.assertCurrentActivity("Wrong Activity", RootActivity.class);
+
+        viewPager = solo.getCurrentActivity().findViewById(R.id.view_pager);
+        bottomNav = solo.getCurrentActivity().findViewById(R.id.bottom_navigation);
+    }
+
+    /**
+     * Tests that the activity starts on the right page

```

Comment:
This is already tested for the login activity

Reply:
this tests that the correct viewpager page shows up *and* also tests the bottom navigation is at the right position. They are similar tests but not the same. No such thing as too much testing :P

Reply:
are we still using user2 for testing or are we using the intentTestingUser profile?

Reply:
i suppooseeee